mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -194,6 +232,285 @@ final class Word2Vec @Since("1.4.0") (
 
   @Since("1.4.1")
   override def copy(extra: ParamMap): Word2Vec = defaultCopy(extra)
+
+  /**
+   * Similar to InitUnigramTable in the original code. Instead of using an array of size 100 million
+   * like the original, we size it to be 20 times the vocabulary size.
```

Comment:
why 20 times the vocabulary size? Also, this comment is misplaced. It belongs with the ``unigramTableSizeFactor``